src/tokens/hooks.ts

```diff
@@ -67,6 +70,15 @@ export function useTokenInfo(tokenId: string) {
   return tokens[tokenId]
 }
 
+/**
+ * @deprecated only used in the old CICO send bar and should be removed when that is updated to multi-chain support
+ */
+export function useTokenInfoBySymbol(symbol: string) {
```

Comment:
I'm not seeing any usages of this anymore, can we just get rid of it? Also, as written it looks like it includes ethereum tokens with no feature flag gating that...

Reply:
Removed in https://github.com/valora-inc/wallet/pull/4242/commits/b5235c47a65fcd2b31f2ba79ad547854a651039c!